### PR TITLE
New package: walledin-1.0

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

File: srcpkgs/walledin/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'walledin'
+pkgname=walledin
```
Comment (abenson) [inline srcpkgs/walledin/template]: Why not make the pkgname WalledIn

Comment (travankor) [reply srcpkgs/walledin/template]: the binary installed is called `walledin`

btw do you know what the arm gcc errors are?

Comment (abenson) [reply srcpkgs/walledin/template]: I think project name trumps binary name -- 

it doesn't look like it used the right gcc for the command. should've been the ${XBPS_CROSS_TRIPLET}-gcc

**Comment by newbluemoon** [comment]
The `Makefile` invokes `gcc` directly, but should make use of the `$CC` variable. You could change it like this, e.g.:

```
pre_build() {
	sed -i -e 's/gcc/$(CC)/' Makefile
}
```

**Pull request closed without merging**